common/djangoapps/enrollment/views.py

```diff
@@ -634,7 +642,15 @@ def post(self, request):
                 }
             )
         except CourseEnrollmentExistsError as error:
-            return Response(data=error.enrollment)
+            return Response(
+                status=status.HTTP_200_OK,
+                data={
+                    "message": (
+                        u"An enrollment for user {username}' in course '{course_id} already exist."
```

Comment:
`An enrollment already exists for user [{username}] in course run [{course_id}].`